<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>小新的串口助手</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>pic/favicon (1).ico</normaloff>pic/favicon (1).ico</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="1">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="styleSheet">
          <string notr="true">background-image: url(:/pic/bg.jpg);
color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 0, 0, 255), stop:0.1875 rgba(255, 255, 255, 255));</string>
         </property>
         <property name="title">
          <string/>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="styleSheet">
             <string notr="true">border-image: url(:/images/dino.png);</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_7">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>139</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_6">
            <property name="font">
             <font>
              <pointsize>18</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-image: url(:/pic/bg.jpg);
color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 0, 0, 255), stop:0.1875 rgba(255, 255, 255, 255));</string>
            </property>
            <property name="text">
             <string>自制串口助手</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_6">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>139</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QCheckBox" name="sendHexBox">
           <property name="styleSheet">
            <string notr="true">background-image: url(:/pic/bg.jpg);
color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 0, 0, 255), stop:0.1875 rgba(255, 255, 255, 255));</string>
           </property>
           <property name="text">
            <string>Hex发送</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="sendEdit">
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>9</pointsize>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">border-image: url(:/pic/true.jfif);
font: 9pt &quot;微软雅黑&quot;;
</string>
           </property>
           <property name="cursorPosition">
            <number>0</number>
           </property>
           <property name="cursorMoveStyle">
            <enum>Qt::CursorMoveStyle::LogicalMoveStyle</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="recvHexBox">
           <property name="styleSheet">
            <string notr="true">background-image: url(:/pic/bg.jpg);
color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 0, 0, 255), stop:0.1875 rgba(255, 255, 255, 255));</string>
           </property>
           <property name="text">
            <string>Hex显示</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="openButton">
           <property name="styleSheet">
            <string notr="true">background-image: url(:/pic/bg.jpg);
color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 0, 0, 255), stop:0.1875 rgba(255, 255, 255, 255));</string>
           </property>
           <property name="text">
            <string>打开</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>28</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="closeButton">
           <property name="styleSheet">
            <string notr="true">background-image: url(:/pic/bg.jpg);
color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 0, 0, 255), stop:0.1875 rgba(255, 255, 255, 255));</string>
           </property>
           <property name="text">
            <string>关闭</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>28</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="sendButton">
           <property name="styleSheet">
            <string notr="true">background-image: url(:/pic/bg.jpg);
color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 0, 0, 255), stop:0.1875 rgba(255, 255, 255, 255));</string>
           </property>
           <property name="text">
            <string>发送</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>28</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="cleanButton">
           <property name="styleSheet">
            <string notr="true">background-image: url(:/pic/bg.jpg);
color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 0, 0, 255), stop:0.1875 rgba(255, 255, 255, 255));</string>
           </property>
           <property name="text">
            <string>清空</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>28</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="saveButton">
           <property name="styleSheet">
            <string notr="true">background-image: url(:/pic/bg.jpg);
color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 0, 0, 255), stop:0.1875 rgba(255, 255, 255, 255));</string>
           </property>
           <property name="text">
            <string>保存文件</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="label_8">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <bold>true</bold>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-image: url(:/pic/bg.jpg);
color: rgb(255, 255, 0);</string>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="styleSheet">
            <string notr="true">background-image: url(:/pic/bg.jpg);
color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 0, 0, 255), stop:0.1875 rgba(255, 255, 255, 255));</string>
           </property>
           <property name="text">
            <string>串口号</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="styleSheet">
            <string notr="true">background-image: url(:/pic/bg.jpg);
color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 0, 0, 255), stop:0.1875 rgba(255, 255, 255, 255));</string>
           </property>
           <property name="text">
            <string>波特率 </string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="styleSheet">
            <string notr="true">background-image: url(:/pic/bg.jpg);
color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 0, 0, 255), stop:0.1875 rgba(255, 255, 255, 255));</string>
           </property>
           <property name="text">
            <string>数据位</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="styleSheet">
            <string notr="true">background-image: url(:/pic/bg.jpg);
color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 0, 0, 255), stop:0.1875 rgba(255, 255, 255, 255));</string>
           </property>
           <property name="text">
            <string>停止位</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="styleSheet">
            <string notr="true">background-image: url(:/pic/bg.jpg);
color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 0, 0, 255), stop:0.1875 rgba(255, 255, 255, 255));</string>
           </property>
           <property name="text">
            <string>校验位</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QComboBox" name="serianumBox">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">border-image: url(:/pic/true.jfif);</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="botolvBox">
           <property name="font">
            <font>
             <bold>true</bold>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>ArrowCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">border-image: url(:/pic/true.jfif);</string>
           </property>
           <property name="editable">
            <bool>true</bool>
           </property>
           <property name="frame">
            <bool>true</bool>
           </property>
           <item>
            <property name="text">
             <string>4800</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>9600</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>115200</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2000000</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Custom</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="databitBox">
           <property name="font">
            <font>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">border-image: url(:/pic/true.jfif);</string>
           </property>
           <item>
            <property name="text">
             <string>8</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>5</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>6</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>7</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="stopbitBox">
           <property name="font">
            <font>
             <bold>true</bold>
            </font>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="styleSheet">
            <string notr="true">border-image: url(:/pic/true.jfif);</string>
           </property>
           <property name="frame">
            <bool>true</bool>
           </property>
           <item>
            <property name="text">
             <string>1</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>1.5</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="jiaoyanBox">
           <property name="font">
            <font>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">border-image: url(:/pic/true.jfif);</string>
           </property>
           <item>
            <property name="text">
             <string>0</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>1</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QPlainTextEdit" name="recvTextEdit">
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">background-image: url(:/pic/bg.jpg);</string>
       </property>
       <property name="lineWidth">
        <number>4</number>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOn</enum>
       </property>
       <property name="tabChangesFocus">
        <bool>false</bool>
       </property>
       <property name="undoRedoEnabled">
        <bool>false</bool>
       </property>
       <property name="readOnly">
        <bool>false</bool>
       </property>
       <property name="plainText">
        <string/>
       </property>
       <property name="cursorWidth">
        <number>3</number>
       </property>
       <property name="maximumBlockCount">
        <number>0</number>
       </property>
       <property name="backgroundVisible">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>140</y>
     <width>176</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/pic/true.jfif);
</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="2" column="1">
     <widget class="QPushButton" name="quitButton">
      <property name="text">
       <string>quit</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QLineEdit" name="botolvEdit">
      <property name="styleSheet">
       <string notr="true">background-image: url(:/pic/true.jfif);</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="okButton">
      <property name="text">
       <string>ok</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="label_7">
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::FocusPolicy::WheelFocus</enum>
      </property>
      <property name="text">
       <string>请输入波特率</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>recvTextEdit</zorder>
  <zorder>label_8</zorder>
  <zorder>groupBox_2</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
